backend/graphql/resolvers/User.js

```diff
@@ -34,16 +24,20 @@ const userResolvers = {
 				throw new Error(
 					'No first name, last name, email, or toggles provided.'
```

Comment:
Same ID issue as the other resolver. Note the need for ID in the thrown error, or remove it from the `if` logic check.